### PR TITLE
Improve the description of LandOnCondition of Aircraft

### PR DESCRIPTION
**Pull request by jrb0001**
This addresses the feedback from @pchote after https://github.com/OpenRA/OpenRA/pull/13434 was merged.

### CODE REVIEW

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
`OpenRA.Mods.Common/Traits/Air/Aircraft.cs:L104: [SP2100] Code line must be no longer than 180 characters (now 197).`

**Comment by jrb0001** [comment]
Fixed.

**Comment by abcdefg30** [comment]
It looks like nothing changed...?

**Comment by jrb0001** [comment]
Forgot to add the file, it seems.

File: OpenRA.Mods.Common/Traits/Air/Aircraft.cs [hunk 1]
```diff
@@ -101,7 +101,8 @@ IEnumerable<object> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, ActorP
 			yield return new FacingInit(PreviewFacing);
 		}
 
-		[Desc("Condition when this aircraft should land as soon as possible and refuse to take off.")]
+		[Desc("Condition when this aircraft should land as soon as possible and refuse to take off. "
+			+ "This only applies while the aircraft is above terrain which is listed in LandableTerrainTypes.")]
```
Comment (atlimit8) [inline OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: nit: the convention is to use a comma since the constructor takes a `param` list of `string`s.

Comment (jrb0001) [inline OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: fixed.

**Comment by atlimit8** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b5c033729bd93041fb514722afc89097b5c3d598...7adfeebb9afdd34dc6285885e47d8487cdc59faa)

**Pull request merged**